### PR TITLE
[Roman Numerals] add practice exercise (v2)

### PR DESCRIPTION
**Pull request by colinleach**
Let's try this again, on a new day.

For the week of [02-06]. The `example.R` doesn't exactly showcase R's best features, but it's the best I've come up with so far.

### CODE REVIEW

**Comment by colinleach** [comment]
I have some draft approaches ready to PR, but I'll wait until this exercise is merged so I have a base for adding the `.approaches` directory.

**Review by jonmcalder — APPROVED** [review]

**Comment by colinleach** [comment]
Once the exercise is merged I can rebase the approaches and PR them. There might be better solutions in there, that can be copied across later.

**Comment by jonmcalder** [comment]
I've just skimmed through the approaches in your branch which look good - thanks. I'm sure we can get those merged in over the weekend ahead of next week's #48in24

**Pull request merged**
